### PR TITLE
fix: improper pointer actions 🛠

### LINKED ISSUE #1762: Bug: Improper pointer actions on user profile page 🪲
### What happened?

- The user's profile will have set of social links which are linked to icons, during hover some of the icons have pointing cursor some of have not that.
- This imbalance is a severe drawback in terms of User Experience.

### Add screenshots

https://github.com/priyankarpal/ProjectsHut/assets/92252895/6e1a549f-b75b-4b26-b421-77773795150a

### What browsers are you seeing the problem on?

Firefox

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Related Issue
Close #1762

## Description
- This PR is about fixing the improper pointer for the social icons on Users Profile.
- This Bug is fixed by adding `cursor-pointer` tailwind utility class so that we explicitly making the pointers actions same for all icons

## Screenshots
https://github.com/priyankarpal/ProjectsHut/assets/92252895/3630f09f-6619-47b8-bd44-afffce33c7b2

### CODE REVIEW

**Comment by 0xabdulkhalid** [comment]
Hey there @priyankarpal 👋🏻 

Please make sure to review this PR.

Thank you!

File: components/ProjectCard.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ProjectCard: NextPage<ProjectCardType> = ({ project }) => {
           href={link}
           target="_blank"
           rel="noreferrer"
-          className=" px-4 items-center group flex gap-2 justify-center text-center text-white border w-1/2 border-gray-700  xl:text-[1rem] md:text-[0.8rem] rounded-md py-[0.35rem] "
+          className="cursor-pointer px-4 items-center group flex gap-2 justify-center text-center text-white border w-1/2 border-gray-700  xl:text-[1rem] md:text-[0.8rem] rounded-md py-[0.35rem] "
```
Comment (priyankarpal) [inline components/ProjectCard.tsx:43]: why `cursor-pointer` here?

**Comment by 0xabdulkhalid** [comment]
Can i get any updates @priyankarpal ?

File: components/ProjectCard.tsx (hunk 1 above)
Comment (priyankarpal) [reply components/ProjectCard.tsx:43]: ok

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**